ncm-cron/src/main/perl/cron.pm

```diff
@@ -405,6 +405,13 @@ sub Configure {
         $self->error("Could not refresh cron by running \"svcadm refresh cron\"")
             if $?;
     }
+    
+    # For Linux: as a workarond for a RedHat 5 cron race condition bug: touch /etc/cron.d
+    #            when we are done generating the files. cron(8) will reload as a result
+    if ($osnamee eq "linux") {
```

Comment:
typo in `$osnamee` (double `e`)?
also, there rest of the code has `$osname eq "solaris"` so i'm not sure if the osname on linux is `linux` (if you're not either, you could use `$osname ne "solaris"`).